cmd/secretkeepassxc.go

```diff
@@ -127,10 +128,43 @@ func (c *Config) keePassXCAttributeFunc(entry, attribute string) string {
 	return outputStr
 }
 
+func readPassword(prompt string) (pw []byte, err error) {
+	fd := int(os.Stdin.Fd())
+	if terminal.IsTerminal(fd) {
+		fmt.Fprint(os.Stderr, prompt)
```

Comment:
I think this should write to `os.Stdout` not `os.Stderr`, same for line 136.

Reply:
Sure, I'll go ahead and update it.

Reply:
Fixed